variables.tf

```diff
@@ -69,6 +69,8 @@ variable "worker_pools" {
     operating_system  = optional(string)
     labels            = optional(map(string))
     minSize           = optional(number)
+    secondary_storage = optional(string)
+    entitlement       = optional(string)
```

Comment:
Remove entitlement here since we already have an ocp_entitlement input to use for pools.

Reply:
`+1`